ui/perfherder/alerts/TagsModal.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Button,
+  Form,
+  FormGroup,
+  Label,
+  CustomInput,
+} from 'reactstrap';
+
+export default class TagsModal extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      tags: [],
+    };
+  }
+
+  componentDidMount = async () => {
+    const { alertSummary, performanceTags } = this.props;
+    const activeTags = alertSummary.performance_tags || [];
+
+    const tags = performanceTags.map((tag) => ({
+      ...tag,
+      active: activeTags.includes(tag.name),
+    }));
+
+    this.setState({
+      tags,
+    });
+  };
+
+  toggleTag = (index) => {
+    const { tags } = this.state;
+
+    tags[index].active = !tags[index].active;
+    this.setState({ tags });
+  };
+
+  listTags = () => {
+    return this.state.tags.map((tag, index) => {
+      return (
+        <CustomInput
+          key={tag.name}
+          type="checkbox"
+          id={tag.name}
+          name={tag.name}
+          label={tag.name}
+          checked={tag.active}
+          onChange={() => this.toggleTag(index)}
+        />
+      );
+    });
+  };
+
+  getActiveTags = () => {
+    const { tags } = this.state;
+    const activeTags = tags.filter((tag) => tag.active);
+
+    return activeTags.map((tag) => tag.name);
+  };
+
+  toggleModal = () => {
+    const { toggle, alertSummary, performanceTags } = this.props;
+    const activeTags = alertSummary.performance_tags || [];
+    const tags = performanceTags.map((tag) => ({
+      ...tag,
+      active: activeTags.includes(tag.name),
+    }));
+
+    this.setState({
+      tags,
+    });
+
+    toggle();
+  };
+
+  render() {
+    const { showModal, updateAndClose } = this.props;
+
+    return (
+      <Modal isOpen={showModal}>
+        <ModalHeader toggle={this.toggleModal}>Alert Tags</ModalHeader>
+        <Form>
+          <ModalBody>
+            <Label for="performanceTags" />
+            <FormGroup>{this.listTags()}</FormGroup>
+          </ModalBody>
+          <ModalFooter>
+            <Button
+              color="secondary"
```

Comment:
You should use "darker-secondary" instead: https://treeherder.readthedocs.io/accessibility.html#colors